rules/game_process.tex

```diff
@@ -158,13 +158,24 @@ \subsubsection{Indirect Kick - Champions Cup}
     A deliberate play at the ball includes successfully kicking the ball, dribbling the ball or the goalkeeper playing at the ball with its hands.
   \item If an indirect free kick is kicked directly into the opponent's goal, the goal does not count and a \texttt{Goal kick} is awarded.
   \item Note that an own-goal may always be scored without requiring an indirect kick.
-  \item A direct kick is allowed for the duration of the penalty kick, but this rule still applies after this kick. A contact in the penalty kick counts among this rule.
 \end{itemize}
 
+There are two exceptions to this rule:
+
+\begin{itemize}
+  \item A direct kick is allowed for the duration of the penalty kick, but this rule still applies after this kick.
```

Comment:
A contact in the penalty kick counts among this rule.

Reply:
If you mean that the second sentence in line 161- should be replicated here, I'll respond immediately because I already know the answer: that concept is expressed in line 173+, with the intention that the rules after this bullet list apply to both types of direct kick.